src/model/Model_CurrencyHistory.cpp

```diff
@@ -85,41 +85,68 @@ int Model_CurrencyHistory::addUpdate(const int& currencyID, const wxDate& date,
 /** Return the rate for a specific currency in a specific day*/
 double Model_CurrencyHistory::getDayRate(const int& currencyID, const wxString& DateISO)
```

Comment:
we should add here overloaded version to simplify code:
`double Model_CurrencyHistory::getDayRate(const int& currencyID, const wxDateTime& Date = wxDateTime::Today())`

Then remove all `const wxString today = wxDateTime::Today().FormatISODate();` and simply call `getDayRate(currencyID)`

Reply:
Agree